src/Host.cpp

```diff
@@ -191,6 +191,7 @@ Host::Host(int port, const QString& hostname, const QString& login, const QStrin
 Host::~Host()
 {
     mIsGoingDown = true;
```

Comment:
`mIsGoingDown` is used by the same name in `mudlet` and seems to be for designating an application-wide close.  In `Host` I honestly didn't look too far, I figured it was closely related to the member in `mudlet` and left it alone since `mIsClosingDown` was not used and seemed fitting for this purpose.